packages/ui/src/FileCard/index.tsx

```diff
@@ -152,15 +146,15 @@ export const FileCard = ({
         <div className="file-details-wrapper">
           <div className="file-name-description-details-wrapper">
             <div>
-              {visibleFileDetailsObject.filename ? (
+              {visibleFileDetailsMap.filename ? (
                 <span className="file-name">{file.filename}</span>
               ) : null}
-              {file.size && visibleFileDetailsObject.filesize && (
+              {file.size && visibleFileDetailsMap.filesize && (
```

Comment:
`filesize` -> `fileSize` typo here?

Reply:
Not actually, field name is written similarly (filename, filesize).